store/db_test.go

```diff
@@ -78,9 +78,10 @@ func createTable(db *DB, t *testing.T) {
 }
 
 func TestDBRace(t *testing.T) {
-	// TODO(sgotti) this will not find concurrenct accesses to ql db from
+	// TODO(sgotti) this will not find concurrent accesses to ql db from
 	// multiple processes using multiple goroutines. A test that spawns at
 	// least two processes using multiple goroutines is needed.
+	// See https://github.com/coreos/rkt/pull/2391
```

Comment:
Should you link to the specific comment?
https://github.com/coreos/rkt/pull/2391#discussion_r59004642